src/tempo.py

```diff
@@ -33,14 +33,15 @@ def __init__(
         self.otlp_grpc_port = 4317
         self.otlp_http_port = 4318
         self.zipkin_port = 9411
-
+        self.jaeger_port = 14268
         self._local_hostname = local_host
 
         self._supported_ingesters: Tuple[RawIngester, ...] = (
             ("tempo", self.tempo_port),
             ("otlp_grpc", self.otlp_grpc_port),
             ("otlp_http", self.otlp_http_port),
             ("zipkin", self.zipkin_port),
+            {"jaeger", self.jaeger_port},
```

Comment:
make this a tuple and the tests should pass

Reply:
also, I'd call this `jaeger_thrift` to distinguish it from `jaeger_grpc`. Speaking off, mind adding that one too, so we're complete?